go.mod

```diff
@@ -46,7 +46,7 @@ require (
 	github.com/pingcap/errors v0.11.5-0.20211224045212-9687c2b0f87c
 	github.com/pingcap/failpoint v0.0.0-20220423142525-ae43b7f4e5c3
 	github.com/pingcap/fn v0.0.0-20200306044125-d5540d389059
-	github.com/pingcap/kvproto v0.0.0-20220711062932-08b02befd813
+	github.com/pingcap/kvproto v0.0.0-20220804022843-f006036b1277
```

Comment:
Because in this commit https://github.com/pingcap/kvproto/commit/f006036b1277bf25e075244992fb55f9f5949917 , two fields are added into the S3 message .  This PR will use the newly added fields to develop this feature.  So I updated the kvproto version to include this commit.